tests/unit/test_power_grid_model.py

```diff
@@ -69,6 +70,51 @@ def input_row():
     }
 
 
+@pytest.fixture
+def input_col_cpp():
```

Comment:
please rename to something more elaborate or delete

Reply:
Addressed.